circuits/src/main.nr

```diff
@@ -1,13 +1,8 @@
-mod padding;
-
-// TODO: Change to final value (<= BLOCK_SIZE - 2).
-global INPUT_SIZE: Field = 10;
-
-// TODO: Change to final value (1088)
-global BLOCK_SIZE: Field = 10; // Blocks are 136 bytes. 138 * 8 = 1088 bits.
+use dep::keccak::permutations as permutations;
+use dep::keccak::constants as constants;
```

Comment:
Why the `as _` renaming to the same name? This should be the default in noir to match rust - do we not support it?

Reply:
Hmm. Just tested it again and it worked so yeah, we can remove this. Reasonably sure that when I was first testing imports I ran into issues unless I did the full path again on any usage of the imported functions but that might have been user error.